contract/src/lib.rs

```diff
@@ -11,8 +13,9 @@ static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 mod nep21;
 mod util;
 
-// Prepaid gas for making a single simple call.
-const SINGLE_CALL_GAS: u64 = 200_000_000_000_000;
+// Prepaid gas costs. TODO: we need to adjust this value properly.
+const SINGLE_CALL_GAS: u64 = 200_000_000_000_000; // for making a single simple call.
+const TX_NEP21_GAS: u64 = 1_000_000_000_000_000_000; // 0.000_001 NEAR
```

Comment:
This will fail, MAX_GAS is 200Tgas per contract call.  Evgeny confimed on discord. `const SINGLE_CALL_GAS: u64` has already the max. 
> 200Tgas per contract call, 300Tgas total per transaction
@Evgeny | NEAR#3816

Reply:
Thanks, will update.